plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py

```diff
@@ -445,7 +447,7 @@ def _export_organisation(self):
                 **self.sia_405_base_common(row, "organisation"),
                 # --- organisation ---
                 auid=row.uid,
-                bemerkung=self.truncate(self.emptystr_to_null(row.remark), 80),
+                bemerkung=self.truncate(self.emptystr_to_null(row.remark), 255),
```

Comment:
As we have now the constraints in the datemodel

https://github.com/teksi/wastewater/blob/42cbbec329a57f5e436c1328d2359c0261fb3763/datamodel/changelogs/0001/03_tww_db_dss.sql#L172

I am not sure if we still need the truncate here. If yes the correction is correct.